var/spack/repos/builtin/packages/py-mkdocs/package.py

```diff
@@ -16,17 +16,30 @@ class PyMkdocs(PythonPackage):
 
     license("BSD-2-Clause")
 
+    version("1.5.3", sha256="eb7c99214dcb945313ba30426c2451b735992c73c2e10838f76d09e39ff4d0e2")
     version("1.3.1", sha256="a41a2ff25ce3bbacc953f9844ba07d106233cd76c88bac1f59cb1564ac0d87ed")
 
+    depends_on("python@3.7:", when="@1.4.1:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-hatchling", when="@1.4.1:", type="build")
     depends_on("py-setuptools", type="build")
```

Comment:
Okay, I kept it in because I know the older versions use it and I wasn't sure if spack wanted to phase out old versions over time or not. Will remove :+1: